website/src/app/kb/quickstart/readme.mdx

```diff
@@ -13,35 +23,38 @@ with the some of the most common configuration options.
   className="mx-auto shadow rounded"
 />
 
+This quickstart guide illustrates a simple Firezone setup. Firezone supports
+more complex deployments than the diagram illustrated above, but by starting
+with the basics laid out here you'll get a better sense for how to deploy
+Firezone for your specific use case.
+
+You'll be able to change everything covered in this guide later, so don't worry
+if you make a mistake or two.
+
 ## Prerequisites
 
 - Firezone account (Don't have an account?
   [Request early access](/product/early-access))
 - Resource you want to give users secure access to (e.g. prod server, database,
-  SaaS application, subnet, etc...)
-- Server or VM (with the ability to deploy a docker container or Linux
-  application) that can connect to both the resource, and the internet
+  SaaS application, or subnet)
+- Server or VM you're able to deploy a Docker container or Linux binary on that
+  can connect to both the Resource and the internet
 
 ## Summary
 
 1. **Sign in to your Firezone Admin Portal** (e.g.
```

Comment:
"Admin Portal" is capitalized here because it's title case, and not because "Admin Portal" is generally capitalized, right?